shared/pingpong/pingpong.go

```diff
@@ -320,7 +319,27 @@ func constructTxn(from, to string, fee, amt uint64, client libgoal.Client, cfg P
 	crypto.RandBytes(noteField[tagLen:])
 
 	// Construct payment transaction
-	txn, err = client.ConstructPayment(from, to, fee, amt, noteField[:], "", [32]byte{}, 0, 0)
+	if cfg.NumAsset == 0 {
+		txn, err = client.ConstructPayment(from, to, fee, amt, noteField[:], "", [32]byte{}, 0, 0)
+		if !cfg.Quiet {
+			fmt.Fprintf(os.Stdout, "Sending %d : %s -> %s\n", amt, from, to)
+		}
+	} else {
+		var assetID uint64
+		for k := range assetParams {
+			assetID = k
+			break
+		}
```

Comment:
We want to pick a random assetID. this code would produce "undefined" assetID, but that id will remain the same across multiple iterations.